taffybar.rc

```diff
@@ -40,8 +40,15 @@ style "taffybar-workspace-border-urgent" = "taffybar-default" {
 widget "Taffybar*" style "taffybar-default"
 widget "Taffybar*WindowSwitcher*label" style "taffybar-active-window"
 widget "*NotificationCloseButton" style "taffybar-notification-button"
-widget "*Workspace-*-active*" style "taffybar-workspace-border-active"
-widget "*Workspace-*-visible*" style "taffybar-workspace-border-visible"
-widget "*Workspace-*-empty*" style "taffybar-workspace-border-empty"
-widget "*Workspace-*-hidden*" style "taffybar-workspace-border-hidden"
-widget "*Workspace-*-urgent*" style "taffybar-workspace-border-urgent"
+
+widget "*Workspace-Border-*-active*"     style "taffybar-workspace-border-active"
+widget "*Workspace-Border-*-visible*"    style "taffybar-workspace-border-visible"
+widget "*Workspace-Border-*-empty*"      style "taffybar-workspace-border-empty"
+widget "*Workspace-Border-*-hidden*"     style "taffybar-workspace-border-hidden"
+widget "*Workspace-Border-*-urgent*"     style "taffybar-workspace-border-urgent"
+
+widget "*Workspace-Container-*-active*"  style "taffybar-default"
+widget "*Workspace-Container-*-visible*" style "taffybar-default"
+widget "*Workspace-Container-*-empty*"   style "taffybar-default"
+widget "*Workspace-Container-*-hidden*"  style "taffybar-default"
+widget "*Workspace-Container-*-urgent*"  style "taffybar-default"
```

Comment:
necessary so that the inside of the button is styled as the rest of taffybar, instead of like the border.